File: csharp/two-fer/TwoFerTests.cs
using Xunit;

public class TwoFerTests
{
    [Fact]
    public void No_name_given()
    {
        Assert.Equal("One for you, one for me.", TwoFer.Speak());
    }

    [Fact]
    public void A_name_given()
    {
        Assert.Equal("One for Alice, one for me.", TwoFer.Speak("Alice"));
    }

    [Fact]
    public void Another_name_given()
    {
        Assert.Equal("One for Bob, one for me.", TwoFer.Speak("Bob"));
    }
}

File: fsharp/two-fer/TwoFerTests.fs
module TwoFerTests

open FsUnit.Xunit
open Xunit

open TwoFer

[<Fact>]
let ``No name given`` () =
    twoFer None |> should equal "One for you, one for me."

[<Fact>]
let ``A name given`` () =
    twoFer (Some "Alice") |> should equal "One for Alice, one for me."

[<Fact>]
let ``Another name given`` () =
    twoFer (Some "Bob") |> should equal "One for Bob, one for me."

